webpack.config.js

```diff
@@ -0,0 +1,28 @@
+var path = require('path');
+var webpack = webpack = require('webpack');
+
+var IS_PRODUCTION = process.env.NODE_ENV == 'production';
+
+module.exports = {
+  output: {
+    path: path.resolve(__dirname, "build", "web", "js"),
+    filename: "[name].js",
+  },
+  devtool: IS_PRODUCTION ? 'source-map' : 'inline-source-map',
+  entry: {
+    "beforePyret": './src/web/js/beforePyret.js',
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
+      'process.env.PYRET': JSON.stringify(process.env.PYRET),
+      'process.env.CURRENT_PYRET_RELEASE': JSON.stringify(process.env.CURRENT_PYRET_RELEASE),
+    }),
+  ].concat(IS_PRODUCTION ? [
```

Comment:
What should I expect if I put `NODE_ENV="production"` in my `.env` and rebuild (after a `make clean`)?

I get "The page has a redirect loop" after restarting the server and reloading the page, but I'm not sure if I'm missing anything/etc.